multiqc/modules/mapdamage/mapdamage.py

```diff
@@ -0,0 +1,282 @@
+""" MultiQC module to parse base misincorporation output from mapdamage2 """
+
+
+import logging
+import os
+from collections import OrderedDict
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import linegraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """mapDamage module"""
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="mapDamage",
+            anchor="mapdamage",
+            href="https://github.com/ginolhac/mapDamage",
+            info="mapDamage: tracking and quantifying damage patterns in ancient DNA sequences.",
+            doi="https://doi.org/10.1093/bioinformatics/btt193",
+        )
+
+        # Init empty dictionaries
+        self.threepGtoAfreq_data = dict()
+        self.fivepCtoTfreq_data = dict()
+        self.lgdist_fw_data = dict()
+        self.lgdist_rv_data = dict()
+
+        # Find and load log files
+        for f in self.find_log_files("mapdamage", filehandles=True):
+            self.parseInput(f)
```

Comment:
I would stick to the conventional pythonic naming style, which is lowercase_underscore. And for MultiQC (somewhat loose) [convention](https://multiqc.info/docs/development/modules/#step-2---parse-data-from-the-input-files), would be good to call the method `parse_logs()`:

```suggestion
            self.parse_logs(f)
```